collector/stat.go

```diff
@@ -98,7 +101,11 @@ func (c *statCollector) Update(ch chan<- prometheus.Metric) (err error) {
 			}
 			// Only some of these may be present, depending on kernel version.
 			cpuFields := []string{"user", "nice", "system", "idle", "iowait", "irq", "softirq", "steal", "guest"}
-			for i, v := range parts[1 : len(cpuFields)+1] {
+			exFieldNum := len(cpuFields)+1
```

Comment:
Ahhh, ok... too many potential words starting with "ex" - this one didn't cross my mind :)

Maybe just "numFields"?